x-pack/legacy/plugins/uptime/public/components/functional/ping_list/doc_link_body.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { EuiIcon, EuiLink, EuiText } from '@elastic/eui';
+
+const bodyDocsLink =
+  'https://www.elastic.co/guide/en/beats/heartbeat/current/configuration-heartbeat-options.html#monitor-http-response';
+
+export const DocLinkForBody = () => {
+  const docsLink = (
+    <EuiLink href={bodyDocsLink} target="_blank">
+      {i18n.translate('xpack.uptime.monitorList.geoName.helpLinkAnnotation', {
+        defaultMessage: 'Docs',
```

Comment:
this should be lower case in the context in which it's used. The text right now says "Read our Docs", it should be "Read our docs"

Reply:
Agreed